### PR TITLE
Add make tool build

### PR DESCRIPTION
**Pull request by JellyBellyDev**
[feat/make-tool-build 41c43c2] Add make tool build

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+init:
```
Comment (garak) [inline Makefile]: I would add following lines on top:

```
%:
        @:

args = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
```

that is useful if an user calls `make` without arguments (it shows all available arguments)

and then
```
.PHONY: init run start stop install install_prod [...] # please complete this list...
```

See [SO answer](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile)

Comment (JellyBellyDev) [reply Makefile]: the first three lines doesn't work :S

Comment (garak) [reply Makefile]: are you using a tab before `@`? I think that you can't use spaces to indent a Makefile

Comment (JellyBellyDev) [reply Makefile]: yep! 
Schermata 2020-03-24 alle 15 21 48

**Pull request merged**